app/javascript/components/rooms/FeatureTabs.jsx

```diff
@@ -4,20 +4,30 @@ import RoomRecordings from '../recordings/RoomRecordings';
 import Presentation from './Presentation';
 import SharedAccess from '../shared_accesses/SharedAccess';
 import RoomSettings from '../room_settings/RoomSettings';
+import useSiteSetting from '../../hooks/queries/site_settings/useSiteSetting';
 
 export default function FeatureTabs() {
+  const { data: shareRoomEnabled } = useSiteSetting('ShareRooms');
```

Comment:
I believe that `useSiteSetting` will be needed on the `EditSiteSetting` form in the admin panel and it should return a list of all site settings with their `:name` and `:value`.